widget/embedded/src/hooks/useSwapInput.ts

```diff
@@ -112,15 +127,34 @@ export function useSwapInput(): UseSwapInput {
       }
       fetchQuote(requestBody)
         .then((res) => {
+          const sortQuotes = sortRoutesBy(sortStrategy, res.results);
+          const quote = sortQuotes.length
+            ? {
+                requestAmount: res.requestAmount,
+                ...sortQuotes[0],
+              }
+            : null;
```

Comment:
Could you assign the appropriate type to prevent errors:
const quote: SelectedQuote | null

Reply:
fixed